### DISCUSSION TITLE
Adding support of conda installation

### DISCUSSION THREAD

**Discussion by frgfm** [pypdfium2-team/pypdfium2#77] [Ideas]
Hi there :wave: 

Really glad to find this project! So far I've been using the pypi installation method, but I was wondering whether there was any plan to add a conda build?

Thanks!

**Comment by mara004** [comment]
Hello again,
I'm afraid I haven't used conda yet. However, if you can point me at some good guide or introduction on how to publish a package on conda, I can look into it.

**Reply by mara004** [reply]
I read about conda a bit and found [this packaging documentation](https://docs.conda.io/projects/conda-build/en/latest/user-guide/tutorials/build-pkgs.html), but I think it's too complex for me. I suppose it would be easier with a pure-python package, but pypdfium2 interfaces with a C library and thus requires platform-specific builds, which makes things more difficult. Maybe a third-party developer can maintain a conda package some day, but I probably won't be able to do it. Sorry.

**Reply by frgfm** [reply]
Hey there :wave: 

Yes to be honest, I intended to first publish the conda build for you and let you know. But I opened this issue because I realized that there ere C bindings and that's out of my expertise on the conda side :sweat_smile: 

I know it's doable but perhaps if we find relevant resources for this later, we can come back to this?

**Reply by mara004** [reply]
Thanks for the update. I agree we can look into this again at some point.

**Reply by mara004** [reply]
Linking https://github.com/mindee/doctr/issues/113 and https://github.com/conda/conda-build/issues/2611#issuecomment-1239338538 for reference

**Reply by mara004** [reply]
Also linking https://github.com/AnacondaRecipes/pypdfium2-feedstock/issues/1

**Reply by mara004** [reply]
Done.
